File: template-rest-assured-java/templates/src/test/java/{{package_name}}/SuiteTest.java
package {{package_name}};

import {{package_name}}.tests.CreateUserTests;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({
        CreateUserTests.class
})
public class SuiteTest {

}

File: template-rest-assured-kotlin/templates/src/test/kotlin/{{package_name}}/SuiteTest.kt
package {{package_name}}

import org.junit.runner.RunWith
import org.junit.runners.Suite
import org.junit.runners.Suite.SuiteClasses
import {{package_name}}.tests.CreateUserTests

@RunWith(Suite::class)
@SuiteClasses(CreateUserTests::class)
class SuiteTest 